This fixes errors like the following (because the cross-compiler is named differently):
    /bin/sh: arm-eabi-ld: not found
    /bin/sh: arm-eabi-gcc: not found

diff --git a/Makefile b/Makefile
index da2108a..52b5d12 100644
--- a/Makefile
+++ b/Makefile
@@ -195,7 +195,7 @@ SUBARCH := arm
 SUBARCH := arm
 export KBUILD_BUILDHOST := $(SUBARCH)
 ARCH		?= $(SUBARCH)
-CROSS_COMPILE	?= /home/niko/arm-eabi-4.7/bin/arm-eabi-
+CROSS_COMPILE	?= $(CONFIG_CROSS_COMPILE:"%"=%)
 CROSS_COMPILE	?= $(CONFIG_CROSS_COMPILE:"%"=%)
 
 # Architecture as present in compile.h
